src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+const operate = (num1, num2, operation) => {
+  let result;
+  const a = Big(num1);
+  const b = Big(num2);
+  switch (operation) {
+    case '+':
+      result = a.plus(b);
+      break;
+    case '-':
+      result = a.minus(b);
+      break;
+    case 'X':
+      result = a.times(b);
+      break;
+    case '/':
+      result = a.div(b);
+      break;
+    default:
+      break;
+  }
+  return parseFloat(result);
+};
+
```

Comment:
Operate.js should have a case for **%** according to no 14 of the project requirements. Here is the link:
https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/